src/app/test/test-folder-tree/test-folder-tree.component.ts

```diff
@@ -20,7 +20,7 @@ export class TestFolderTreeComponent {
     folderBehaviourOnClick: 'select',
     expandAllFolders: true,
     highlightOpenFolders: false,
-    autoSelectCondition: (item: CreateTreeItem) => this.autoSelectItem(item),
+    autoOpenCondition: (item: CreateTreeItem) => this.autoSelectItem(item),
```

Comment:
Why did you change this? autoOpenCondition is not the same as autoSelectCondition